### PR TITLE
fix(ccdep-2469): Trigger builds + use latest

### PR DESCRIPTION
**Pull request by foster36**
- Trigger builds for new `node` + `typescript` packages
- Use latest packages in `node` + `react` packages

### CODE REVIEW

File: typescript/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
 
    "parserOptions": {
       "ecmaVersion": 2017,
-      "lib": ["ES2017"]
+      "lib": ["ES2017"],
```
Comment (ontehfritz) [inline typescript/index.js:7]: Bump this to ES2022 :)

**Review by ontehfritz — CHANGES_REQUESTED** [review]

File: typescript/index.js (hunk 1 above)
Comment (foster36) [reply typescript/index.js:7]: JS in on 2017 as well, should we bump both packages?

Comment (chris-skoblenick) [reply typescript/index.js:7]: I think so?

Comment (foster36) [reply typescript/index.js:7]: Not sure if the [TS docs](https://www.typescriptlang.org/tsconfig#lib) are just out-of-date or not but they're listing 2021 as the latest supported version.

**Review by chris-skoblenick — APPROVED** [review]

**Review by ontehfritz — APPROVED** [review]

**Pull request merged**